test/jdk/java/awt/Frame/MultiScreenTest.java

```diff
@@ -99,21 +97,22 @@ public List<JFrame> init() {
         for (int j = 0; j < gs.length; j++) {
             GraphicsConfiguration[] gc = gs[j].getConfigurations();
             if (gc.length > 0) {
-                for (int i = 0; i < gc.length / 2; i++) {
-                    JFrame f = new JFrame(gc[i]); //test JFrame( gc )
-                    GCCanvas c = new GCCanvas(gc[i]);//test canvas( gc )
-                    Rectangle gcBounds = gc[i].getBounds(); //test getBounds()
+                for (int i = 0; i < gc.length && i < 10; i++) {
+                    JFrame f = new JFrame(gc[i]); // test JFrame( gc )
+                    GCCanvas c = new GCCanvas(gc[i]); // test canvas( gc )
+                    Rectangle gcBounds = gc[i].getBounds(); // test getBounds()
                     int xoffs = gcBounds.x;
                     int yoffs = gcBounds.y;
 
                     f.getContentPane().add(c);
                     f.setTitle("Screen# " + Integer.toString(j) + ", GC#" + Integer.toString(i));
                     f.setSize(300, 200);
-                    f.setLocation(400 + xoffs, (i * 150) + yoffs);//test displaying in right location
+                    f.setLocation(400 + xoffs, (i * 150) + yoffs); // test
+                    // displaying in right location
                     list.add(f);
 
                     Frame ditherfs = new Frame("DitherTest GC#" + Integer.toString(i), gc[i]);
-                    ditherfs.setLayout(new BorderLayout()); //showDitherTest
+                    ditherfs.setLayout(new BorderLayout()); // showDitherTest
```

Comment:
The comment is misleading: setting a layout doesn't show anything.
```suggestion
                    ditherfs.setLayout(new BorderLayout());
```